src/Mod/Draft/InitGui.py

```diff
@@ -147,6 +147,7 @@ def Deactivated(self):
         Log("Draft workbench deactivated\n")
 
     def ContextMenu(self, recipient):
+        from DraftGui import translate
```

Comment:
By the way, importing the `translate` function from `DraftGui` means that we have to process the entire `DraftGui` module. We don't really want to do this if we just want the `translate` function.

This is the reason I split the `translate` function into its own module, `draftutils.translate`, so we can import this component alone without pulling the rest of `DraftGui`. Perhaps we can test if this works correctly.